html2docx/html2docx.py

```diff
@@ -101,6 +103,38 @@ def finish_p(self) -> None:
             self.r.text = self.r.text.rstrip()
         self._reset()
 
+    def init_table(self, attrs: List[Tuple[str, Optional[str]]]) -> None:
+        if self.table_cell is not None:
+            table = self.table_cell.add_table(rows=0, cols=0)
+        else:
+            table = self.doc.add_table(rows=0, cols=0)
+        self.tables.append((table, -1, -1))
+
+    def finish_table(self) -> None:
+        table = self.tables.pop()[0]
+        section = self.doc.sections[0]
+        page_width = section.page_width - section.left_margin - section.right_margin
+        page_width = int(page_width * (0.5 ** len(self.tables)))
+        for col in table.columns:
+            col.width = page_width // len(table.columns)
+
+    def init_tr(self) -> None:
+        table, row, col = self.tables[-1]
+        row += 1
+        col = -1
+        self.tables[-1] = (table, row, col)
+        table.add_row()
+
+    def init_tdth(self) -> None:
+        table, row, col = self.tables[-1]
+        col += 1
+        self.tables[-1] = (table, row, col)
+        if col >= len(table.columns):
+            table.add_column(0)
```

Comment:
A comment explaining that columns must have a width to form a valid docx would be very helpful. It could also explain that the column width is computed in `finish_table`, because the total column count is needed to evenly spread the width.